app/models/user.rb

```diff
@@ -5,5 +5,7 @@ class User < ApplicationRecord
 
   has_many :identities, dependent: :destroy
 
-  validates :email, presence: true
+  def to_s
```

Comment:
don't rewrite `to_s` please, because `User.first #=> "User Userovich"` and we need object id

`def full_name` much more better